README.md

```diff
@@ -2,24 +2,18 @@
 
 Bosun is a time series alerting framework developed by Stack Exchange. Scollector is a metric collection agent. Learn more at [bosun.org](http://bosun.org).
 
-[![Build Status](https://travis-ci.org/bosun-monitor/bosun.svg?branch=master)](https://travis-ci.org/bosun-monitor/bosun/branches)
+[![Build Status](https://travis-ci.org/bosun-monitor/bosun.svg?branch=master)](https://travis-ci.org/bosun-monitor/bosun)
```

Comment:
What's the reason for changing the build status badge ?

Reply:
Thanks, that was accidentally committed. Fixed and rebased the branch on the latest master.